frigate/track/norfair_tracker.py

```diff
@@ -230,7 +237,16 @@ def match_and_update(self, frame_time, detections):
                 )
             )
 
-        tracked_objects = self.tracker.update(detections=norfair_detections)
+        coord_transformations = None
+
+        if self.ptz_autotracker_enabled:
+            coord_transformations = self.ptz_motion_estimator.motion_estimator(
+                detections, frame_time, self.camera_name
+            )
+
+        tracked_objects = self.tracker.update(
```

Comment:
I am assuming but just wanted to double check that `None` is the default and this doesn't affect the tracker in other use casees?

Reply:
Yep. 

https://tryolabs.github.io/norfair/2.2/reference/tracker/#norfair.tracker.Tracker.update